### PR TITLE
Feature/mob 4369 remove payment cell

### PR DESCRIPTION
**Pull request by BartlomiejSopalaKarhoo**
## JIRA
[MOB-4369](https://karhoo.atlassian.net/browse/MOB-4369)

## DESCRIPTION
Removed old presenters, views etc. Part of logic moved to Checkout

## Types of changes

What types of changes does your code introduce to the project?
_Put an `x` in the boxes that apply_

- [ ] Bugfix 
- [ X] New feature 
- [ ] Tech Debt
- [ ] Breaking change 
- [ ] Documentation Update
- [ ] Other: *define what type of change it is*

## CHECKLIST

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. This is simply a reminder of what we are going to look for before merging code._
Please review the guidelines for contributing to this repository.

- [ ] All work relating to the ticket is complete (Code complete/Acceptance Criteria met)
- [ ] Lint and unit tests pass locally (if appropriate)
- [ ] Added tests that prove the fix is effective or that feature works (if appropriate)
- [ ] Documentation (if appropriate)

## SCREENSHOTS
[Add screenshots if any for changes you have made to the App/UI]

### CODE REVIEW

File: KarhooUISDK/Extensions/UIColor+Extensions.swift [hunk 1]
```diff
@@ -85,4 +86,14 @@ extension UIColor {
             return UIColor(cgColor: properColor)
         */
     }
+
+    func getColor() -> Color {
+        return Color(
```
[the remark this answers is not in the document]
Comment (BartlomiejSopalaKarhoo) [reply KarhooUISDK/Extensions/UIColor+Extensions.swift:91]: It is fixed in previous PR, will disappear once I merge it.

File: KarhooUISDK/Screens/CheckoutScreen/Checkout/KarhooCheckoutViewController.swift [hunk 2]
```diff
@@ -363,9 +383,14 @@ final class KarhooCheckoutViewController: UIViewController, CheckoutView {
         bookingButton.setRequestedMode()
     }
     
+    func set(nonce: Nonce){
+        paymentNonce = nonce
+//        presenter.completeBookingFlow()
```
Comment (dianakarh) [inline KarhooUISDK/Screens/CheckoutScreen/Checkout/KarhooCheckoutViewController.swift:388]: missed cleanup

File: KarhooUISDK/Screens/CheckoutScreen/Checkout/KarhooCheckoutViewController.swift [hunk 3]
```diff
@@ -363,9 +383,14 @@ final class KarhooCheckoutViewController: UIViewController, CheckoutView {
...
+    }
+    
     func resetPaymentNonce() {
         paymentNonce = nil
-        passengerDetailsAndPaymentView.noPaymentMethod()
+//        passengerDetailsAndPaymentView.noPaymentMethod()
```
[the remark this answers is not in the document]
Comment (BartlomiejSopalaKarhoo) [reply KarhooUISDK/Screens/CheckoutScreen/Checkout/KarhooCheckoutViewController.swift]: done

File: KarhooUISDK/Screens/CheckoutScreen/Checkout/KarhooCheckoutViewController.swift (hunk 2 above)
Comment (BartlomiejSopalaKarhoo) [reply KarhooUISDK/Screens/CheckoutScreen/Checkout/KarhooCheckoutViewController.swift]: done

**Review by dianakarh — APPROVED** [review]

**Review by AleksanderWedrychowskiKarhoo — APPROVED** [review]

**Pull request merged**